build.sbt

```diff
@@ -178,7 +178,7 @@ lazy val commonSettings = Seq(
   sourceManaged := baseDirectory.value / "src_managed",
   resourceManaged := baseDirectory.value / "resource_managed",
   libraryDependencies ++= Dependencies.common,
-  IntegrationTest / parallelExecution := false,
+  // IntegrationTest / parallelExecution := false, // allow test in parallel
```

Comment:
Thoughts on removing this commented line? There's nothing inherent about integration tests that require non-parallelism, it was really just our CLI tests which happened to be all integration tests, but that's no longer the case anymore.

Reply:
I would rather be explicit about this. Particularly since the CLI testing can't do this.